Updates to Red Beginner Routes & FAQ

SquareSR opened the pull request:
FAQ now includes a link to the Red/Blue Strategy Wiki
Lass/nolass routes have some helpful embedded images and more links to images.
Classic beginner now has links to some gym leader locations and more indicators for where to fly such as (toggle down X) 
Also improved lass/nolass description of Boom Strats to avoid ppl thinking they should use a revive after Weezing. 
Added a note that items should be bought in the order that they are listed in the guide.

SquareSR committed:
Update faq.md

Add Strategy Wiki Link for RB. Change link to Classic to send directly to the guides list.

SquareSR committed:
Update README.md

Adding embedded movement maps for noLass (still in progress)...

SquareSR committed:
Update README.md

SquareSR committed:
Update README.md

SquareSR committed:
Update README.md

merged changes made to no-lass without changing the differences between route 3 and reaching Bill.

SquareSR committed:
Update README.md

Added some links to location of Gym's in addition to now linking the Strategy Wiki guide in the FAQ.

SquareSR commented:
I also moved EQ Menu to inside Safari Zone, since this was found to be optimal and gives more wiggle room on repel steps for beginners. I did NOT move EQ menu for red classic since we use rare candies in EQ menu and it's going to be tricky not having cursor on Nidoking.

SquareSR committed:
Classic Beginner Shopping Note

 **always buy items in the order that they are listed in this guide**. Doing so is both the fastest way to buy the items and sets up our inventory in the correct order.

SquareSR committed:
Add note and link to Ether Route Adjustments

SquareSR committed:
Note Linking Adjustments to Ether Route in Advanced

SquareSR committed:
Move EQ menu inside safari for advanced route

SquareSR committed:
Boom Strats Embedded into Intermediate Route like other HP Strats already are

SquareSR commented:
Overview: Strategy Wiki linked in FAQ, Images added to Red Glitchless Beginner Guides, Note added to both Classic and Glitchless beginner guides about buying items in the correct order, EQ Menu moved to Safari Zone for Glitchless guides, and lastly embedded Boom Strats into the Intermediate route just as other HP Strats already are.

SquareSR committed:
Red WR Route Moved EQ Menu into Safari

Also, replaced imgur for optimal movement on 9 srepel since it was expired/removed.

SquareSR committed:
Link for Victory Road Movement to Lass Route

SquareSR committed:
VR Link to fwr commentary for movement

Grogir commented on docs/gen-1/red-blue/main-glitchless/WR-Route/README.md:
```diff
@@ -352,15 +352,21 @@ Cycling Road
 
 Post Cycling Road Menu:
-- Use Super Repel (skip repeling if we only bought 9)
+- Bike
```
That seems a bit light for a "menu", maybe reword this

Grogir commented on docs/gen-1/red-blue/main-glitchless/beginner-route/lass/README.md:
```diff
@@ -1,15 +1,25 @@
 # Pokemon Red Glitchless Beginner Guide, Lass Version
 
+# Pokemon Red Glitchless Beginner Guide, No-Lass Version
```
to remove

Grogir commented on docs/gen-1/red-blue/main-glitchless/beginner-route/lass/README.md:
```diff
@@ -91,12 +102,14 @@ Options: Battle Style to Set before leaving Brock's gym
 - Weedle: Leer + HA x2
 	- Leer + HA + Tackle if you got String Shot hit OR Tackle will always kill
-- Caterpie: HA x2 + Tackle
+- Caterpie: Leer + HA + Tackle
```
This was HA x2 because pp is supposedly not a problem on lass

SquareSR replied:
This one I am still in favor of changing since I've seen some beginners go from Beginner Lass to No-Lass, but continue to HAx2 the Caterpie not realizing they are wasting an HA. Perhaps giving the option to do either is good I think

Grogir commented on docs/gen-1/red-blue/main-glitchless/beginner-route/lass/README.md:
```diff
@@ -450,27 +480,31 @@ Get the Poke Flute and Fly to Celadon City
-- Use the center
-- Bike to Saffron City
+- MUST Take the center in Celadon. Replenishes PP & sets Celadon as the warp point.
+- Bike East to Saffron City, trading Fresh Water for passage, and enter Silph Co.
```
"enter Silph Co" is said twice :nerd:

Grogir commented on docs/gen-1/red-blue/main-glitchless/beginner-route/lass/README.md:
```diff
@@ -707,9 +767,9 @@ After Lance:
-		- Pidgeot: X Acc + HD
+Pidgeot: **Turn 1: USE X SPECIAL**, then adapt:
+	- If Pidgeot does NOT use Sky Attack on turn 1:
+		- Pidgeot: use X Acc on turn 2, HD
```
You broke the formatting, on top of that i'm not sure adding that many words make it clearer

SquareSR replied:
Got it, reverting this back entirely to how it was.

Grogir commented on docs/gen-1/red-blue/main-glitchless/beginner-route/lass/README.md:
```diff
@@ -1,15 +1,25 @@
+	- [Post Hiker Backup Paras](https://pastebin.com/j5gtY4cy) (recommended)
+		- [Easy - Older version](https://pastebin.com/7kcZu3g4) (easier path, but select yoloball)
 	- [Post Nerd Backup Paras](../../resources/postnerd-backup-paras.md)
+	- [Route 3 Moon Manip](https://pastebin.com/tggXpQRC)
```
This was not listed on the lass document because its simply not usable with lass, i'm fine letting this here but hopefully it doesnt cause any confusion

SquareSR replied:
Oh that makes sense, I agree. Copy-paste error here on my part, will remove it.

Grogir commented on docs/gen-1/red-blue/main-glitchless/beginner-route/nolass/README.md:
```diff
@@ -129,7 +139,7 @@ Catch a flyer:
 ## Mount Moon
 
-- Do not learn a full Moon Manip for your first runs, start by learning [Post Hiker Backup Paras](https://pastebin.com/j5gtY4cy) (or the [older version](https://pastebin.com/7kcZu3g4)) and walk through the first floor normally ([movement map](https://i.imgur.com/xzgmNBk.jpeg)). You will choose later if you want to learn [Route 3 Moon](https://pastebin.com/tggXpQRC) (optimal for this route) or [En ...
+- Do not learn a full Moon Manip for your first runs, start by learning [Post Hiker Backup Paras](https://pastebin.com/j5gtY4cy) (or the easier [older version](https://pastebin.com/7kcZu3g4)) and walk through the first floor normally ([movement map](https://i.imgur.com/xzgmNBk.jpeg)). You will choose later if you want to learn [Route 3 Moon](https://pastebin.com/tggXpQRC) (optimal for this route) ...
```
Why is this comment different on both files, should be copied

SquareSR replied:
Understood - will make wording match except for the disclaimers regarding Rt.3 and Entr needing to differ.

Grogir commented on docs/gen-1/red-blue/main-glitchless/beginner-route/lass/README.md:
```diff
@@ -630,16 +686,20 @@ Menu after leaving the gym:
 - Pick up the [hidden Max Ether](https://gunnermaniac.com/pokeworld?map=34#8/90) from the water
-- Use a Super Repel one step into the cave
-- Watch a top leaderboard run for how to do the Strength boulder puzzles
+- Exit water, super repel, and get on the bike.
```
Detail but would appreciate upper case Super Repel

Grogir commented:
Personally not a fan of adding so many boom strats tables in the intermediate guide, but im not going to force my view

SquareSR committed:
wr route remove post-cycling road menu

SquareSR committed:
Misc. Quick Fixes to No-Lass Route

SquareSR committed:
Fixed typo in Intermediate Route

SquareSR committed:
Update boom strats 7-11 for hp-strats.md

On the current routes we will always be out of blizzards even if we hit them all. Always will Elix on either Rhydon or sooner, never after Rhydon.
1 Eggs, 2 Tangela, 3 CT's Rhyhorn, 4 Machop, and 5 Gio's Rhyhorn

SquareSR committed:
Race Route EQ Menu inside Safari Zone

SquareSR committed:
Update hp-strats-for-races.md

Add in more detailed Backup boom strats for the HP Strats for Races

SquareSR committed:
Capitalize Super Repel in nolass for consistent formatting

SquareSR committed:
Also Revert Champion Fight back for nolass, formatting fixed

SquareSR committed:
undo update 7-11 boom stats, given that PPUP routes may delay Elix

SquareSR committed:
Delay Elixer if 25+ After Blackbelt note now included.
